### ISSUE TITLE
Reporting shows only total amount of deducted time

### ISSUE THREAD

**Issue by Does-This-Work** [Keleo/DeductionTimeBundle#8]
It would be great, if under Reporting the deducted time, would also be displayed under the day.
Right now I can only see how much time I spend in total on breaks but I cannot see, how long my break was for a certain day.

|  | Working Hours Total | Day 1 | Day 2 | Day 3 | Day 4 | Day 5 |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| Working  | 40  | 9  | 7 | 8 | 8 | 8 | 
| Breaks  | -5  |
| Working Hours Total | 35  | 8| 6| 7| 7| 7|

**Comment by kevinpapst** [comment]
What screen is that?

**Comment by Does-This-Work** [comment]
The screen is located under 
/reporting/user/week
/reporting/user/month
/reporting/user/year
![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/086119cb-1c76-43d8-8caa-4c375aec8ff4)

**Comment by kevinpapst** [comment]
<img width="973" alt="Bildschirmfoto 2023-05-25 um 10 37 46" src="https://github.com/Keleo/DeductionTimeBundle/assets/533162/87e4121d-6265-4273-9e71-747b380fce40">

Maybe you have to upgrade to the latest Kimai release?

**Comment by Does-This-Work** [comment]
Running Kimai: 2.0.23
Plugin Version: 2.0.1

Anything that I maybe misconfigured with the plugin? This is how I configured the **break** activity
![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/2b4deba3-feda-4a2d-828a-e395b5cb9e3f)


User View
![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/766393a7-ee26-40c3-9726-c09e159457e3)

Admin View
![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/03ea2e7c-0927-4e86-bfef-945c306fd187)

**Comment by kevinpapst** [comment]
Strange... I can't see any cause for that, sorry 🤷

**Comment by Does-This-Work** [comment]
Okay! Thanks for taking the time, to look at this issue

**Comment by kevinpapst** [comment]
Can you share the actual timesheet as well please. What you show doesn't really make sense. 
Afair there is no check that hides negative times in this screen, so I do not understand how this can happen.

**Comment by Does-This-Work** [comment]
Of course! I added some extra data. I also saw, that I made a mistake in the earlier screenshot.
I hope the new screenshots can tell you more

![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/f6dbb633-2f0a-4dd3-a5cf-428b7906f7c8)

![grafik](https://github.com/Keleo/DeductionTimeBundle/assets/102190830/178af013-86b9-40cc-bdaf-9572a2b763c1)

Thank you for your help! 👍

EDIT: I also tried reinstalling the plugin but that didn't do anything...

**Comment by stephan-senkbeil** [comment]
I just came across the same issue using this plugin.
I tracked this down to the twig template for the reports in which activities with a negative duration are excluded (https://github.com/kimai/kimai/blob/e98732f44a75b60a53a9a55e5a9d1b890e473526/templates/reporting/report_by_user_data.html.twig#L106).

Changing `column.duration > 0` to e.g. `column.duration != 0` fixes this issue but I haven't checked if anything else breaks with this change as I'm not at all familiar with the Kimai code base.
